plugins/ca-python-standard/bin/ca-python-install

```diff
@@ -40,3 +40,19 @@ fi
 poetry -v install || poetry -v install || exit 1
 echo "Devbox setup has completed poetry install"
 pre-commit install --install-hooks || echo "WARNING: could not install pre-commit hooks" >&2
+
+# hook env vars up to cert files for agreeable netskope x python packages
+if [ -d "$HOME" ]; then
+  curl -o $HOME/Downloads/netskope_bundle.pem https://raw.githubusercontent.com/cultureamp/devbox-extras/main/certs/nscacert_combined.pem
```

Comment:
**suggestion** (may be obsolete now anyway but: ) the `virtenv` folder would probably be a better place to put the cert